### ISSUE TITLE
[experimental tree-sitter]: Template literal interpolation characters without style

### ISSUE THREAD

**Issue by confused-Techie** [pulsar-edit/pulsar#562]
### Thanks in advance for your bug report!

- [X] Have you reproduced issue in [safe mode](https://pulsar-edit.dev/docs/atom-archive/faq/sections/what-does-safe-mode-do.html)?
- [X] Have you used the [debugging guide](https://pulsar-edit.dev/docs/atom-archive/hacking-atom/#debugging) to try to resolve the issue?
- [X] Have you checked our [FAQs](https://pulsar-edit.dev/docs/launch-manual/sections/faq/) to make sure your question isn't answered there?
- [X] Have you checked to make sure your issue does not already [exist](https://github.com/issues?q=is%3Aissue+user%3Apulsar-edit+is%3Aopen)?
- [X] Have you checked you are on the latest release of Pulsar?

### What happened?

When creating a template literal on the new experimental Tree-Sitter branch, template literal interpolation characters have zero styling, as well as the variable within them. This does cause them to standout, but I'd assume that the interpolation items would receive something.

Now I do want to mention on @savetheclocktower's [new scope list](https://gist.github.com/savetheclocktower/c9607b97477d4817911e4f2f8db89679#file-scope-taxonomy-md) they do actually reference this scope being applied, since it is receiving a scope. So it may make more sense to ensure the default One Dark theme (as well as possible others) are styling this scope correctly. 

But here's some example code:

```js
const val = `Any kind of template literal, with a ${value}`;
```

The opening bracket has the following scopes:
  * source.js
  * string.quoted.template.js
  * meta.mebedded.line.interpolation.js

But looks like this:

![image](https://github.com/pulsar-edit/pulsar/assets/26921489/24c946cb-426d-46fd-a1bb-9d329c520112)

Which for reference here's what the old tree-sitter implementation looked like:

![image](https://github.com/pulsar-edit/pulsar/assets/26921489/f07ba316-a318-48ae-9b16-dc46c17d54bd)

### Pulsar version

1.105.2023052001

### Which OS does this happen on?

🪟 Windows

### OS details

22H2

### Which CPU architecture are you running this on?

64-bit(x86_64)

### What steps are needed to reproduce this?

1. Create a `.js` file
2. Add the example code from above
3. View styling it receives.

**Comment by asiloisad** [comment]
The new scope list is great future, but has started many compability problems. IMO the one-dark and one-light themes must support both scope systems. The modern tree-sitter + one-light + python is not usable now 🙁

**Comment by confused-Techie** [comment]
@asiloisad Please feel free to let us know what breaks within Python, we can't fix it if we don't know what exactly is broken.

But to clarify, yes it is a fantastic feature, but it's also something we have to do. If we want to upgrade to newer Electron and Node versions, then we cannot use non-context aware native modules. Such as Tree-Sitter, in due time the existing Tree-Sitter will have to be removed, with the current experimental Tree-Sitter taking it's place. So we hope to iron out all existing issues before we have to remove the old Tree-Sitter. If you'd like more information on why this is a change that we have to make feel free to read a [blog article on it](https://pulsar-edit.dev/blog/20230209-mauricioszabo-tree-sitter-part-1.html). 

But thanks a ton for letting us know what's gone wrong, appreciate the contribution!

**Comment by savetheclocktower** [comment]
@confused-Techie The fix for this just got pushed to #555.

@asiloisad I'll be happy to fix things that are going wrong with Python right now, whether they're syntax highlighting problems or deeper functionality problems. Feel free to open a bug or to ping me on Discord.
